src/app/[locale]/projects/add/page.tsx

```diff
@@ -7,456 +7,133 @@
 // SPDX-License-Identifier: EPL-2.0
 // License-Filename: LICENSE
 
-'use client'
+"use client"
 
-import styles from '@/css/AddProjects.module.css'
-import AddKeyValueComponent from '@/components/AddKeyValue'
-import SelectCountryComponent from '@/components/SelectCountry'
-import AddAdditionalRolesComponent from '@/components/AddAdditionalRoles'
-import { SearchUsersModal } from '@/components/sw360'
-import Link from 'next/link'
-import { usePathname } from 'next/navigation'
+import { Col, Row, ListGroup, Tab, Button } from 'react-bootstrap'
+import Summary from "./components/Summary/Summary"
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import { useRouter } from 'next/navigation'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { TypeOptions, toast } from 'react-toastify'
+import ApiUtils from '@/utils/api/api.util'
+import ProjectPayload from "@/object-types/CreateProjectPayload"
+import { useState } from 'react'
+import { AUTH_TOKEN } from '@/utils/env'
 
-export default function Summary() {
-    const pathName = usePathname()
+
+export default function AddProjects() {
+
+    const router = useRouter();
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [projectPayload, setProjectPayload] = useState<ProjectPayload>({
+        name: '',
+        description: '',
+        version: '',
+        visibility: '',
+        projectType: '',
+        tag: '',
+        domain: '',
+        leadArchitect: 'test@mail.com',
+        state: 'ACTIVE',
+        phaseOutSince: '',
+        moderators: null,
+        contributors: null
+    });
+
+    const alert = (text: string, type: TypeOptions) =>
+        toast(text, {
+            type,
+            position: toast.POSITION.TOP_LEFT,
+            theme: 'colored',
+        })
+
+    const createProject = async () => {
+        const response = await ApiUtils.POST('projects',
+                                              projectPayload,
+                                              AUTH_TOKEN)
+
+        if (response.status == HttpStatus.CREATED) {
+            const responseData = await response.json()
+            alert(t('Your project is created'), 'success')
+            router.push('/projects')
+        } else {
+            alert(t('There are some errors while creating project'), 'error')
```

Comment:
Missing key **There are some errors while creating project** in language files

Reply:
Added in the language file.